confd/vars/test-vars.yaml

```diff
@@ -10,8 +10,7 @@ kilda_tests_zookeeper_hosts: "localhost"
 
 kilda_tests_opentsdb_hosts: "localhost"
 kilda_tests_opentsdb_port: "4242"
-kilda_tests_victoriametrics_hosts: "localhost"
-kilda_tests_victoriametrics_port: "8428"
```

Comment:
Why did you remove it? We do not want to test Victoria metrics in func tests?

Reply:
I found that we can (and, in fact, we did) use the application variables from main.yaml instead of test-vars.yam.